### PR TITLE
support new elilo multipath config var (fate #317208)

### PR DESCRIPTION
**Pull request by wfeldt**

### CODE REVIEW

File: src/Core/ELILO.pm [hunk 1]
```diff
@@ -172,6 +172,7 @@ sub GetMetaData() {
 	noedd30		=> "bool:Don't force EDD30 Mode if not set:",
 	fpswa		=> "path:Specify the Filename for a specific FPSWA to load:",
 	"secure-boot"	=> "string:Support Secure Boot:auto",
+	multipath	=> "int:Number of additional multipath entries:0",
```
Comment (jreidinger) [inline src/Core/ELILO.pm]: I think this is not what we want as we do not want any additional entries in yast UI and metadata in SLE10 is used for UI :(

Comment (wfeldt) [inline src/Core/ELILO.pm]: yast must be aware of it, else the value vanishes.

Comment (jreidinger) [inline src/Core/ELILO.pm]: hmm, but problem is that it is shown for every user even without mutlipath and also on ia64. let me check code

Comment (wfeldt) [inline src/Core/ELILO.pm]: It still defaults to 0 (or empty). pbl is ok with unknown keys but yast edits them out.

Comment (jreidinger) [inline src/Core/ELILO.pm]: Well, maybe this part is wrong https://github.com/openSUSE/perl-bootloader/blob/master/src/Core/ELILO.pm#L371
I think we should not throw away something we do not know. If we simple pass it thru it should not break anything and also do not remove customer own stuff there.

Comment (wfeldt) [inline src/Core/ELILO.pm]: That's not the place. I never see the 'Ignoring' log line.

Comment (jreidinger) [inline src/Core/ELILO.pm]: that is strange, as pbl should use cached __lines and remove or replace only changed parts, but maybe there is some magic in yast2-bootloader as SLE11 still use quite strange concept of metadata.

**Comment by wfeldt** [comment]
I've changed the type to string because then the default (unset) will not write a 'multipath' line into elilo.conf. Raymund would be ok with this solution.

**Comment by jreidinger** [comment]
@wfeldt and how it look like in yast2-bootloader?

**Comment by wfeldt** [comment]
An empty default.

**Comment by jreidinger** [comment]
@wfeldt and then customer can change it to some kind of string? This doesn't sound much user friendly for me

**Comment by wfeldt** [comment]
The description says 'Number'. Note also that this is an sp4-only option. It will not appear e.g. in sle12.

**Comment by wfeldt** [comment]
It might be good when yast would allow to disable numerical entries. But you end up with <NULL> if you try an empty value (it doesn't check for that - which looks in fact like a real bug).

**Comment by wfeldt** [comment]
End up with &lt;NULL&gt;

**Comment by wfeldt** [comment]
Forget my last comment. It doesn't accept an emtpy _default_.

**Comment by jreidinger** [comment]
@wfeldt OK, so if it is case, then I think numerical entry is better then string one as it at least do not allow to specify something very stupid.

**Comment by wfeldt** [comment]
Yes but the entry should not appear at all as the default.

**Comment by jreidinger** [comment]
@wfeldt I check source code in yast2-bootloader and it really do not allow to not pass value from int selection. That's pretty annoying. I check how hard will be to skip this option.

**Comment by jreidinger** [comment]
It looks like it should be possible to delete it similar as in https://github.com/yast/yast-bootloader/blob/Code-11-SP1/src/generic/global_options_widget.ycp#L41 but I am not sure if it doesn't mean, that we remove such option. Code is quite old and magic.

**Comment by wfeldt** [comment]
I'd rather not.

**Comment by jreidinger** [comment]
@wfeldt ok, so lets keep as it is and lets hope it won't confuse too much customers. LGTM

**Comment by wfeldt** [comment]
ok

**Pull request merged**